lib/api_entity.rb

```diff
@@ -93,6 +93,13 @@ def create!(params = {}, headers = {})
       new parse_jsonapi(response)
     end
 
+    def batch(params = {}, headers = {})
+      request = prepare_json_request(params, headers)
+      batch_path = "#{singular_path.sub(':id', headers[:authorization].split.last)}/batch"
```

Comment:
This doesn't seem right. This is getting the user token, not the subscription ID.